ruler-gradle-plugin/src/main/kotlin/com/spotify/ruler/plugin/apk/ApkSanitizer.kt

```diff
@@ -158,7 +167,16 @@ class ApkSanitizer(
                 entry.name.startsWith("/lib/") -> FileType.NATIVE_LIB
                 else -> FileType.OTHER
             }
-            return AppFile(entry.name, type, entry.downloadSize, entry.installSize)
+            val resourceType: ResourceType? = when {
+                entry.name.startsWith("/res/drawable") -> ResourceType.DRAWABLE
+                entry.name.startsWith("/res/layout") -> ResourceType.LAYOUT
+                entry.name.startsWith("/res/raw") -> ResourceType.RAW
+                entry.name.startsWith("/res/values") -> ResourceType.VALUES
+                entry.name.startsWith("/res/font/") -> ResourceType.FONT
```

Comment:
Is there a reason why we have a trailing slash for fonts but not for all of the others? Is this on purpose?

Reply:
Yeah, Initially I started with a trailing slash on all of them, afterwards, I realized that we can have drawable-hpi, values with a suffix of the language values-fr for example, but for fonts, it should only be font. But for consistency I can remove the trailing slash for the font as well, won't make a difference.

Reply:
Ah, that makes sense. I'd say then let's remove it then for fonts as well.